docs/developers/packaging.md

```diff
@@ -0,0 +1,298 @@
+[official PyPA GitHub Action][4]. This workflow will also create a GitHub Release.
+
+Once the Python package makes it to PyPI, it will be picked by the `conda-forge` bots, which will
+automatically submit a PR to the [`napari-feedstock`][1] repository. This is all automated by the
+`conda-forge` infrastructure, so we only need to check that the metadata in the recipe has been
+adjusted for the new release. Pay special attention to the runtime dependencies and version strings!
+
+Once the conda-forge CI is passing and the PR is approved and merged, the final packages will be
+built on the default branch and uploaded to the `conda-forge` channel. Due to the staging steps and
+CDN synchronization delays, the conda packages can take up to 2h to be available.
+
+### Nightly packages
+
+We also build nightly packages off `main` and publish them to the `napari/label/nightly` channel.
+These are the same packages that are used in the `constructor` installers (see below), so their CI
+is specified in `.github/workflows/bundle_conda.py`.
+
+To do it in a `conda-forge` compatible way, we actually _clone_ `napari-feedstock` and patch the
+source instructions so the code is retrieved from the repository branch directly. The version is
+also patched to match the `setuptools-scm` string. After [rerendering][8] the feedstock, we run
+`conda-build` in the same way `conda-forge` would do and upload the resulting tarballs to our
+Anaconda.org channel.
+
+Additionally, the tarballs are also passed as artifacts to the next stage in the pipeline: building
+the `constructor` installers (more below).
+
+## Installers
+
+Once the packages have been built and uploaded to their corresponding repositories, we can bundle
+them along with their dependencies in a single executable that end users can run to install napari
+on their systems, with no prior knowledge of `pip`, `conda`, virtual environments or anything.
+
+A software installer is usually expected to fulfill these requirements:
+
+* It will install the application so it can be run immediately after.
+* It will provide a convenient way of opening the application, like a shortcut or a menu entry.
+* It will allow the user to uninstall the application, leaving no artifacts behind.
+
+Right now, we are using two ways of generating the installers:
+
+* With `briefcase`, which takes PyPI packages.
+* With `constructor`, which takes `conda` packages.
+
+`conda` packages offer several advantages when it comes to bundling dependencies, since it makes
+very little assumptions about the underlying system installation. As a result, `constructor` bundles
+include libraries that might be missing in the target system and hence should provide a more robust
+user experience.
+
+### Briefcase-based installers
+
+[`briefcase`][5] based installers are marked for deprecation so we will not discuss them here.
+If you are curious, you can check `bundle.py` and `.github/workflows/make_bundle.yml` for
```

Comment:
Can we make those links. `[bundle.py](some link)`

Reply:
We might need to remove this after all if the Briefcase based installers are going to be deprecated. Needs to be decided!

Reply:
probably makes sense to remove after we switch away from briefcase. @tlambert03?